Lib/test/test_pathlib.py

```diff
@@ -182,8 +177,7 @@ class _BasePurePathTest(object):
             ('', 'a', 'b'), ('a', '', 'b'), ('a', 'b', ''),
             ],
         '/b/c/d': [
-            ('a', '/b/c', 'd'), ('a', '///b//c', 'd/'),
```

Comment:
Is there a specific reason for dropping the `///b//c` test?

Reply:
"///b//c" is an invalid UNC path. The server name is empty, and the share name is "b". It isn't equivalent to "/b/c". 

WinAPI `GetFullPathNameW()`:

```pycon
>>> os.getcwd()
'C:\\Temp'

>>> nt._getfullpathname('///b/c/../..')
'\\\\\\b'
>>> nt._getfullpathname('/b/c/../..')
'C:\\'
```

WinAPI `PathCchSkipRoot()`, the basis for `nt._path_splitroot()`:

```pycon
>>> nt._path_splitroot('///b/c')
('///b/', 'c')
>>> nt._path_splitroot('/b/c')
('/', 'b/c')

```

Since the current behavior of pathlib is wrong, I suppose Barney is dropping the test for now until the correct behavior can be tested. There are two options to fix this, if I recall correctly. Either `ntpath.splitdrive()` can be changed to split "///b" as a drive, or pathlib can handle such cases specially.

Reply:
I found that calling `ntpath.normpath()` on some invalid UNC paths produced different results depending on whether or not Python is running on a Windows system. I logged this here: #96290.

That bug seems to be fairly minor; I'm the only person to spot it, months after it was introduced. The expected behaviour isn't clear-cut.

In this PR, we use `normpath()` to normalize path anchors, which means we're newly affected by the bug. This causes a pathlib test failure, but as the expected behaviour isn't clear, I figured that removing the test is a reasonable way forward. Happy to reconsider!

Reply:
I removed four more assertions (all invalid UNC paths) for the same reason: https://github.com/barneygale/cpython/commit/fbaadf45d443c33a1df8d1122aee0805de3c7822

Reply:
I've proposed a fix (#100351) for #96290, but it's worth noting that it changes the same tests as this PR. As such I don't think it blocks this PR.